pkgs/development/libraries/kde-frameworks/ki18n.nix

```diff
@@ -14,9 +14,5 @@ mkDerivation {
   propagatedNativeBuildInputs = [ gettext python ];
   buildInputs = [ qtdeclarative qtscript ];
   patches = [
```

Comment:
Why not remove patches completely?

Reply:
Feels silly to have code churn around boilerplate that is likely to come back

Reply:
Okay :) It's silly, but much cleaner :D